src/entities/purchase-params.ts

```diff
@@ -21,4 +22,26 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * Optional customization for the translations used in the purchase flow.
+   * The keys are the locale codes and the values are objects with the same keys as the default translations.
+   */
+  customTranslations?: CustomTranslations;
+
+  /**
+   * The locale to use for the purchase flow. If not specified, the device's locale will be used.
+   */
+  selectedLocale?: string;
+
+  /**
+   * Specifies if the locale should be picked automatically using the navigator.language property.
+   * Defaults to false.
+   */
+  useBrowserLocale?: boolean;
```

Comment:
Hmm I'm wondering whether we need to support this... If the developer wants to enable this, they can just pass the browser locale to `selectedLocale` I think? Seems like an unneeded API though I'm likely missing some use case 😅

Reply:
Removed